apis/src/components/molecules/live_timer.rs

```diff
@@ -139,34 +111,40 @@ pub fn LiveTimer(side: Signal<Color>) -> impl IntoView {
                     api.game_check_time(&game_id);
                 }
             }
-        }
-    });
-
-    create_effect(move |_| {
-        if user_needs_warning() {
-            sounds.play_sound(SoundType::LowTime);
-            timer_signals.low_time.issued.set(true);
-        }
-    });
-    create_effect(move |_| {
-        if should_refresh_warning() {
-            timer_signals.low_time.issued.set(false);
-        }
-    });
+        },
+        //Has immediate = true, hence not unused
+        WatchOptions::default().immediate(true),
+    );
+    #[allow(unused)]
+    whenever_with_options(
+        user_needs_warning,
+        move |_, _, issued| {
+            let issued = issued.unwrap_or_default();
+            if issued {
+                !should_refresh_warning()
+            } else {
+                sounds.play_sound(SoundType::LowTime);
+                true
```

Comment:
Why does this closure need to return something? can't the condition here be modified, to me it seems like it doesn't need two branches

Reply:
the return from the closure is the "issued" flag now instead of getting it from the signal